app/src/main/java/com/sdpteam/connectout/profileList/EventParticipantsListActivity.java

```diff
@@ -2,30 +2,37 @@
 
 import static com.sdpteam.connectout.event.viewer.EventActivity.PASSED_ID_KEY;
 
-import androidx.appcompat.app.AppCompatActivity;
-import androidx.fragment.app.Fragment;
-
-import android.os.Bundle;
-
 import com.sdpteam.connectout.R;
 import com.sdpteam.connectout.utils.WithFragmentActivity;
 
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import androidx.appcompat.widget.Toolbar;
+import androidx.fragment.app.Fragment;
+
 /**
  * List view of the participants of an event
- * */
+ */
 public class EventParticipantsListActivity extends WithFragmentActivity {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         //Create the view of the activity.
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_container);
+        setContentView(R.layout.activity_user_list);
+
+        Toolbar toolbar = findViewById(R.id.user_list_toolbar);
+        Button filterButton = findViewById(R.id.user_list_button);
+        setSupportActionBar(toolbar);
+        toolbar.setNavigationOnClickListener(v -> this.finish());
+        filterButton.setVisibility(View.GONE);
```

Comment:
Good job with the xml reuse, this might be useful for the profile list refactoring!

Reply:
yes!